pathfinding_service/cli.py

```diff
@@ -52,6 +55,22 @@ def get_default_registry_and_start_block(
         sys.exit(1)
 
 
+def setup_logging(log_level: str, log_config: TextIO):
+    """ Set log level and (optionally) detailed JSON logging config """
+    level = getattr(logging, log_level)
+    logging.basicConfig(
+        level=level,
+        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+        datefmt='%m-%d %H:%M:%S',
+    )
+    logging.getLogger('web3').setLevel(level)
```

Comment:
I misread the `ERROR` log level as `INFO` when changing the code. What would be a good default behaviour in you opinion? Keep `urllib3.connectionpool` at `ERROR` unless overwritten in a config file?

I would like to have a sane default behaviour so that only deviations from that have to be specified and most users won't need to provide any log config file.

Reply:
I would leave the special cases for `web3` and `urllib` for now. That behavior can be overwritten by the user in a config file.

Reply:
On the other hand I just tried without the special casing and the only warning I got were:
```
01-07 14:28:43 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:43 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:43 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:28:44 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
01-07 14:29:05 - urllib3.connectionpool - WARNING - Connection pool is full, discarding connection: parity.ropsten.ethnodes.brainbot.com
```
So maybe this isn't needed at all.

Reply:
I removes the special cases in 4a6a5179f5dbc5da8434602ec17f89000796de40.